src/material-experimental/mdc-list/selection-list.spec.ts

```diff
@@ -566,14 +653,6 @@ describe('MDC-based MatSelectionList without forms', () => {
       expect(listItemEl.componentInstance.value).toBe(componentFixture.componentInstance.itemValue);
     });
 
-    it('should pick up indirect descendant lines', () => {
```

Comment:
Note: This test has been removed because the indirect descendant scenario never really worked within the legacy MDC list and also with the MDC-based one this does not work. The use-case does not seem worth it (given that supporting it would require upstream changes in framework as `ng-content` does not support indirect descendants / or some local hacks) 

```html
<mat-list-item>
  <div>
    <span matListItemTitle>..</span>
    ...
  </div>
</mat-list-item>
```